tests/api2/test_audit.py

```diff
@@ -259,7 +259,7 @@ def test_token_login():
                         "credentials_data": {
                             "parent": {
                                 "credentials": "LOGIN_PASSWORD",
-                                "credentials_data": {"username": "root"},
+                                "credentials_data": {"username": "root", "account_attributes": ["LOCAL", "SYS_ADMIN"]},
```

Comment:
Do we need this in the audit logs?

Reply:
I'm kind of torn, there won't be many of these, but if you want me to I can remove from the credentials.dump() output.